src/panoramic/cli/mapper/model.py

```diff
@@ -36,7 +36,7 @@ def map_model_join_from_local(join: PanoModelJoin) -> ModelJoin:
     )
 
 
-def map_field_from_remote(field: ModelField) -> PanoModelField:
```

Comment:
Not a big fan of the import path `src/panoramic/cli/mapper/model.py`. Can we go with `src/panoramic/cli/model_mapper.py` (preferred) or `src/panoramic/cli/model/mapper.py` ?

Reply:
Same for `src/panoramic/cli/mapper/field.py`

Reply:
Sure, no strong feelings on that one.

Reply:
Changed in 0e99fd9 and e3d739b.